src/js/vic-v2/config/form.js

```diff
@@ -160,17 +162,53 @@ const formConfig = {
           title: 'Photo upload',
           uiSchema: {
             'ui:title': 'Upload Your Photo',
-            photo: {
-              'ui:field': PhotoField,
-              'ui:title': 'Please upload a current photo of yourself that’ll appear on your Veteran ID Card.',
+            photo: _.assign(fileUploadUI('Select files to upload', {
+              endpoint: '/v0/vic/profile_photo_attachments',
+              fileTypes: [
+                'png',
+                'tiff',
+                'tif',
+                'jpeg',
+                'jpg',
+                'bmp'
+              ],
+              maxSize: 26214400,
```

Comment:
Should we store these magic numbers as constants somewhere?

Reply:
Yes even if it's just naming them at the top of this file it makes their purpose / constraint / type clearer to maintainers.

Reply:
Thanks for the helpful idea @rianfowler, I've updated.